SConstruct

```diff
@@ -375,6 +375,8 @@ for name, path in modules_detected.items():
     else:
         enabled = False
 
+    opts.Add(BoolVariable("module_" + name + "_enabled", "Enable module '%s'" % (name,), enabled))
```

Comment:
For context, I moved this so that the `module_<name>_enabled` option would come before the options specific to that module in `scons --help`. It shouldn't impact any of the logic otherwise (unless some custom modules rely on this being undefined in `get_opts`, but I doubt it :)).